src/interactive.jl

```diff
@@ -33,8 +33,8 @@ routes, task, server = interactive_server(Website.asset_paths()) do
     )
 end
 
-# Once everything looks goo, export the static site
-dir = joinpath(@__DIR__, "docs")
+# Once everything looks good, export the static site
+dir = @path joinpath(@__DIR__, "docs")
```

Comment:
I think this isn't needed?

Reply:
oh, perhaps not, if it's just for documentation.  i just simply `grep`'ed for all instances of `__DIR__` and `__FILE__ `and prefixed with `@path`.  i can delete if you like.  but the bigger for me is why does the Smiley MWE above not work?  thanks for the review!